Cpix/Internal/XmlElements.cs

```diff
@@ -213,6 +230,46 @@ public sealed class HlsSignalingDataElement
 	}
 	#endregion
 
+	#region Content key periods
+	[XmlRoot("ContentKeyPeriod", Namespace = Constants.CpixNamespace)]
+	public sealed class ContentKeyPeriodElement
+	{
+		[XmlAttribute("id")]
+		public string Id { get; set; }
+
+		[XmlIgnore]
+		public int? Index { get; set; }
+
+		[XmlAttribute("index")]
+		public string IndexAsXmlString
+		{
+			get => Index?.ToString();
+			set => Index = value != null ? (int?)int.Parse(value) : null;
+		}
+
+		[XmlIgnore]
+		public DateTime? Start { get; set; }
+
+		[XmlAttribute("start")]
+		public string StartAsXmlString
+		{
+			get => Start?.ToString("o"); 
```

Comment:
What's the significance of "o" here? Perhaps a comment can explain

Reply:
This converts the time to the standard "ISO 8601" format. This complies with the XSD "datetime" format and is otherwise also widely used.